build.sbt

```diff
@@ -27,7 +27,7 @@ lazy val microservice = Project(appName, file("."))
       "-Ywarn-unused",
       "-Xlint",
       "-feature",
-      "-Werror",
+      //"-Werror",
```

Comment:
Do we want to leave this commented?

Reply:
No. I'll fix the warnings and put the flag back